config/splat.us.stwrp.yaml

```diff
@@ -54,7 +54,7 @@ segments:
       - [0x1228, .data, e_layout] # layout entries data
       - [0x1420, cmp, D_80181420]
       - [0x1764, cmp, D_80181764]
-      - [0x1D08, cmp, D_80181D08]
+      - [0x1D08, raw, D_80181D08]
```

Comment:
It turns out this isn't cmp?

Reply:
Yeah, the game performs a raw copy of that potion of data. It represents CLUT data and it's not compressed